intranet/apps/users/views.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=consider-using-with
```

Comment:
Do we want to change this file too?

Reply:
No, the logic wouldn't work very well in that file. It's not a straightforward conversion there.